### PR TITLE
Docs release notes 11 7 24

### PR DESCRIPTION
**Pull request by paperclypse**
Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

File: src/content/docs/release-notes/docs-release-notes/docs-11-08-2024.mdx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+### Major changes
+
+* Reorganized the infrastructure docs category and aligned URLs with their location in the repository. Some behind-the-scenes work that makes it easier to find and work on these docs.
+* A significant rewrite of [.NET agent API](/docs/apm/agents/net-agent/net-agent-api/net-agent-api/) removed obsolete information, reorganized some content, updated example code, added more detail about what the APIs do, added a link to our sample code repo, and added ID tags to the distributed tracing APIs.
+
+### Minor changes
+
+* Added DynamoDB to [.NET agent compatibility and requirements](/docs/apm/agents/net-agent/getting-started/net-agent-compatibility-requirements/#database-core).
+* The tag `hostStatus: shutdown` prevents a loss of signal incident from being opened. We updated [create NRQL alert conditions](/docs/alerts/create-alert/create-alert-condition/create-nrql-alert-conditions/) and [loss of signal and gap filling NerdGraph tutorial](/docs/apis/nerdgraph/examples/nerdgraph-api-loss-signal-gap-filling/).
```
Comment (paperclypse) [inline src/content/docs/release-notes/docs-release-notes/docs-11-08-2024.mdx:20]:
```suggestion
* The tag `hostStatus: shutdown` prevents the opening of a loss of signal incident. We updated [create NRQL alert conditions](/docs/alerts/create-alert/create-alert-condition/create-nrql-alert-conditions/) and [loss of signal and gap filling NerdGraph tutorial](/docs/apis/nerdgraph/examples/nerdgraph-api-loss-signal-gap-filling/).
```

File: src/content/docs/release-notes/docs-release-notes/docs-11-08-2024.mdx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+---
+
+### New docs
+
+* [FOSSA integration](/docs/vulnerability-management/integrations/fossa/) describes how this new integration supports your application security initiatives by showing exploitable vulnerabilities in your open source libraries.
+
+
+### Major changes
+
+* Reorganized the infrastructure docs category and aligned URLs with their location in the repository. Some behind-the-scenes work that makes it easier to find and work on these docs.
```
Comment (nbaenam) [inline src/content/docs/release-notes/docs-release-notes/docs-11-08-2024.mdx:14]: As a suggestion, I'd add the link to the infra docs

File: src/content/docs/release-notes/docs-release-notes/docs-11-08-2024.mdx [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+* Added DynamoDB to [.NET agent compatibility and requirements](/docs/apm/agents/net-agent/getting-started/net-agent-compatibility-requirements/#database-core).
+* The tag `hostStatus: shutdown` prevents the opening of a loss of signal incident. We updated [create NRQL alert conditions](/docs/alerts/create-alert/create-alert-condition/create-nrql-alert-conditions/) and [loss of signal and gap filling NerdGraph tutorial](/docs/apis/nerdgraph/examples/nerdgraph-api-loss-signal-gap-filling/).
+* Updated [default infrastructure monitoring data](/docs/infrastructure/infrastructure-data/default-infra-data/) with our improved data aggregation.
+* Added a [deprecation notice for Kubernetes](/eol/2024/10/deprecation-notice-v1.26-and-lower/).
+* Improved readability in [create infrastructure "host not reporting" conditions](/docs/infrastructure/infrastructure-alerts/create-infrastructure-host-not-reporting-condition/).
+* In [layerless instrumentation](/docs/serverless-function-monitoring/aws-lambda-monitoring/instrument-lambda-function/enable-serverless-monitoring-aws-lambda-layerless/), added examples for the ES Module and new steps for installing the the Extension Only Layer.
```
Comment (nbaenam) [inline src/content/docs/release-notes/docs-release-notes/docs-11-08-2024.mdx:24]: There are 2 `the` here.

**Review by nbaenam — APPROVED** [review]
Hi @paperclypse, I just made some small suggestions! Feel free to take them or leave them🙂

File: src/content/docs/release-notes/docs-release-notes/docs-11-08-2024.mdx (hunk 2 above)
Comment (paperclypse) [reply src/content/docs/release-notes/docs-release-notes/docs-11-08-2024.mdx:14]: I was on the fence about it, but I'll add a link if you think I should :)

**Review by akristen — APPROVED** [review]

**Pull request merged**